scripts/page-file-references-are-correct

```diff
@@ -17,12 +20,12 @@ for page in $(jq -r ".pages[].script" $CONFIGPATH); do
         exit 1
     fi
 
-    page_script_relative_path="pages/$page_basename.js"
-    jq ".web_accessible_resources" $MANIFESTPATH | grep "$page_script_relative_path" >/dev/null
+    # TODO: manifest is no longer manually created, so will need to check in a different way
```

Comment:
TODO is not necessary as the page scripts are automatically added to `web_accessible_resources`